### ISSUE TITLE
Dancer::Plugin::Bcrypt doesn't provide a $VERSION

### ISSUE THREAD

**Issue by willert** [LoonyPandora/Dancer-Plugin-Bcrypt#7]
Dancer warns about uninitialized variables on startup due to this issue, so it is a mild annoyance ;)

Thanks

**Comment by bigpresh** [comment]
Yeah, just saw this myself:

```
Use of uninitialized value $version in concatenation (.) or string at /usr/local/share/perl/5.10.0/Dancer/Handler/Standalone.pm line 58, <DATA> line 16.
>> Dancer::Plugin::Bcrypt ()
```

**Comment by LoonyPandora** [comment]
Fixed in 4340c691c1e95d0caa8e
